solutions/api/src/lambda/token/utils/verifyClientAssertion.ts

```diff
@@ -18,6 +25,35 @@ export const verifyClientAssertion = async (
     );
   }
 
+  if (!iat) {
+    return errorManager.throwError(
+      "invalidClientAssertion",
+      "Missing iat in client assertion",
```

Comment:
could we include the `iss`uer value (as `client_id`) so we know which AMC client has an issue. Could we all include it in the error as an additional object (not concatenated into the message, but in an object) - also could we include the `client_id` as a dimension on to metrics so we can see which client is having specific issues via the DT dashboard

➕ Could we also do this for all other errors please